main.go

```diff
@@ -130,6 +122,8 @@ func updateApps(client *cfclient.Client, watchedApps map[string]*consumer.Consum
 			}()
 
 			watchedApps[app.Guid] = conn
+		} else if newClient {
+			watchedApps[app.Guid].Stream(app.Guid, authToken)
```

Comment:
Does this cause a new stream to be created? Does it discard the new event and error channels returned from this function? What happens to the existing stream?

Sounds like we need a test for `updateApps(…, true)` :thinking: